aiomisc/thread_pool.py

```diff
@@ -52,6 +53,15 @@ def context_partial(func: F, *args: Any, **kwargs: Any) -> Any:
 )
 
 
+class ThreadPoolStatistic(Statistic):
```

Comment:
There is no way to get time distribution. It would be useful to have some histograms as well.

Reply:
In next release I hope.